packages/rmb_direct_client/lib/client.ts

```diff
@@ -103,6 +103,15 @@ class Client {
       await this.createSigner();
       const twinId = await this.tfclient.twins.getTwinIdByAccountId({ accountId: this.signer.address });
       this.twin = await this.tfclient.twins.get({ id: twinId });
+      if (!twinId) {
+        throw new Error({ message: `Couldn't find a user for the provided mnemonic on this network.` });
+      }
+    } catch (e) {
+      console.log(e);
```

Comment:
I believe the console.log here is not needed

Reply:
It might be another error and we show this error to be user friendly but we will need to know the original error in the console as developers